src/visual.ts

```diff
@@ -1556,7 +1561,9 @@ module powerbi.extensibility.visual {
                     d: (link: SankeyDiagramLink) => {
                         return this.getLinkLabelSvgPath(link);
                     },
-                    id: (link: SankeyDiagramLink) => `linkLabelPaths${(link.source.label.name || "").replace(/\W*/g,"")}-${(link.destination.label.name || "").replace(/\W*/g,"")}`
+                    id: (link: SankeyDiagramLink) => {
+                        return SankeyDiagram.createLink(link);
```

Comment:
Change to 
return SankeyDiagram.createLink(link, true);
linkLabelPaths prefix is required